File: 04_arrays/arrays.c
#include <stdio.h>

int main(){

    char *nombres_mascotas[5] = {"Tommy", "Zeus", "Coco", "Toby"};

    for(int i = 0; i < 4; i++){
        printf("%d %s \n",i , nombres_mascotas[i]);
    }
    return 0;
}

File: 04_arrays/arrays.go
package main

import "fmt"

func main() {
    /* Define un arreglo de tamaño 4 de nombre de mascotas*/

    var NombresMascotas = [4]string{"Tommy", "Zeus", "Coco", "Toby"}
    // podemos usar ... en vez de 4.


    /* Recorre todo el arreglo */
    for index, option := range NombresMascotas {
        // range itera sobre los items de un arreglo, 'slice' o 'map' 
        fmt.Println(index, option)
    }
}








